### PR TITLE
Add list services from user endpoint

### PR DESCRIPTION
**Pull request by tomas-stefano**
[Trello Card](https://trello.com/c/52ix822P/1011-all-versions-for-a-given-user)

## Context

Create endpoint that returns all the services for a given user ID.

### CODE REVIEW

File: app/controllers/services_controller.rb [hunk 1]
```diff
@@ -13,4 +13,10 @@ def create
       render json: ErrorsSerializer.new(service).attributes, status: :unprocessable_entity
     end
   end
+
+  def collaborators
```
Comment (brenetic) [inline app/controllers/services_controller.rb:17]: i defo think we need to have the idea of collaborators, however in this instance this method is returning services for an individual user right? 

we probably should have an endpoint that returns a list of collaborators for a given service at some point

File: spec/requests/get_services_from_user_spec.rb [hunk 2]
```diff
@@ -0,0 +1,64 @@
+RSpec.describe 'GET /services/user/:user_id' do
```
Comment (brenetic) [inline spec/requests/get_services_from_user_spec.rb:1]: maybe a bit picky, and this is how it's defined in the docs, but maybe we should have `/services/users/:user_id`? Plural on the `users` to keep it consistent with `versions` and `services`.

File: app/controllers/services_controller.rb (hunk 1 above)
Comment (tomas-stefano) [reply app/controllers/services_controller.rb:17]: Yes, returning services. We had difficulty on the naming 😅

Comment (brenetic) [reply app/controllers/services_controller.rb:17]: ok, maybe `services_for_user` or `user_services`, something like that?

Comment (tomas-stefano) [reply app/controllers/services_controller.rb:17]: `services_from_user` maybe?

Comment (brenetic) [reply app/controllers/services_controller.rb:17]: hmm, the `from` implies the user is sending the services to the metadata api. i'll leave it up to you peeps though 😆

Comment (tomas-stefano) [reply app/controllers/services_controller.rb]: We changed to `services_for_user` to be the same naming as the documentation

File: spec/requests/get_services_from_user_spec.rb (hunk 2 above)
Comment (brenetic) [reply spec/requests/get_services_from_user_spec.rb:1]:
```suggestion
RSpec.describe 'GET /services/users/:user_id' do
```

**Review by brenetic — APPROVED** [review]
🚀

**Pull request merged**